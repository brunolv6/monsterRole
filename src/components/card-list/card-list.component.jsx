import React from 'react';
import { Card } from '../card/card.component.jsx'

import './card-list.style.css'

export const CardList = ({ monster }) => (
    <div className='grid'>
        {
            monster.map(m => (
                <Card key={m.id} id={m.id} name={m.name}/>
            ))
        }
    </div>
)