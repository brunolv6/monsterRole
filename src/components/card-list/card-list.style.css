.grid{
    width: 85vw;
    display: flex;
    flex-wrap: wrap;
    margin: 10px auto;
}