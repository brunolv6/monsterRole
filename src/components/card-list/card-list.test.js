import React from 'react'

import { shallow } from 'enzyme'

import { CardList } from './card-list.component'

import toJson from 'enzyme-to-json';

it('testing the render of cart-list', () => {
    const mockRobot = [
        {
            id: 1,
            name: "Jose",
            username: "jojo",
            email: "jojo@gmail.com"
        }
    ]

    expect(toJson(shallow(<CardList monster={mockRobot}/>))).toMatchSnapshot()
})