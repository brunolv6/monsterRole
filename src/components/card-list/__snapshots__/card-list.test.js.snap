// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing the render of cart-list 1`] = `
<div
  className="grid"
>
  <Card
    id={1}
    key="1"
    name="Jose"
  />
</div>
`;
